adapters/richaudience/richaudience.go

```diff
@@ -179,8 +180,8 @@ func (a *adapter) MakeBids(request *openrtb2.BidRequest, requestData *adapters.R
 				bidType := getMediaType(seatBid.Bid[i].ImpID, reqBid)
 
 				if bidType == "video" {
-					seatBid.Bid[i].W = reqBid.Video.W
-					seatBid.Bid[i].H = reqBid.Video.H
+					seatBid.Bid[i].W = ptrutil.ValueOrDefault(reqBid.Video.W)
+					seatBid.Bid[i].H = ptrutil.ValueOrDefault(reqBid.Video.H)
```

Comment:
Nitpick: Line 110 above indicates that `nil` and `0` values for `W` and `H` are filtered out. Should we simply dereference the pointer that we know is valid as a small optimization?
```
181
182                 if bidType == "video" {
183 -                   seatBid.Bid[i].W = ptrutil.ValueOrDefault(reqBid.Video.W)
    +                   seatBid.Bid[i].W = *reqBid.Video.W
184 -                   seatBid.Bid[i].H = ptrutil.ValueOrDefault(reqBid.Video.H)
    +                   seatBid.Bid[i].H = *reqBid.Video.H
185                 }
186
adapters/richaudience/richaudience.go
```

Reply:
Done.